dashboard/of-cloud-dashboard/handler.js

```diff
@@ -110,13 +128,17 @@ module.exports = (event, context) => {
       headers['Content-Type'] = 'text/html';
 
       const isSignedIn = /openfaas_cloud_token=.*\s*/.test(event.headers.cookie);
+      headers["Set-Cookie"] = event.headers.cookie
+
 
       const { base_href, public_url, pretty_url, query_pretty_url } = process.env;
       content = content.replace(/__BASE_HREF__/g, base_href);
       content = content.replace(/__PUBLIC_URL__/g, public_url);
       content = content.replace(/__PRETTY_URL__/g, pretty_url);
       content = content.replace(/__QUERY_PRETTY_URL__/g, query_pretty_url);
       content = content.replace(/__IS_SIGNED_IN__/g, isSignedIn);
+      content = content.replace(/__ORGANIZATIONS__/g, organizations);
```

Comment:
Is this required still? It was there before because `__ORGANIZATIONS__` was attached to Window, not sure what it will replace now?

Reply:
Weren't we going to remove that? Or was it another variable?